### DISCUSSION TITLE
Need help with bot

### DISCUSSION THREAD

**Discussion by hacker173** [aiogram/aiogram#958] [Q&A]
Hi
Need help with bot
Problem:
After receiving a certain command, check all users of the group for factors - the user joined the group < 1 week, the last message is more than 1 month - delete users who fall under these factors

**Comment by depocoder** [comment]
Just code this command, what the problem?

**Reply by hacker173** [reply]
I do not know exactly how to implement the verification of these factors

**Reply by Olegt0rr** [reply]
Collect the data and check it.
If you're waiting for someone to implement this: use freelance instead.
